person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..100_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age?
+  end
```

Comment:
- [ ] Kindly update this function implementation to match this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=Public%20method%20can_use_services%3F%20that%20returns%20true%20if%20person%20is%20of%20age%20or%20if%20they%20have%20permission%20from%20parents.). The function should return true if a person `is of age` `or` if they `have permission from parents`.